arviz/plots/distcomparisonplot.py

```diff
@@ -26,42 +26,45 @@ def plot_dist_comparison(
     backend_kwargs=None,
     show=None,
 ):
-    """Plot to compare fitted and unfitted distributions.
+    r"""Plot to compare fitted and unfitted distributions.
 
     The resulting plots will show the compared distributions both on
     separate axes (particularly useful when one of them is substantially tighter
-    than another), and plotted together, so three plots per distribution
+    than another), and plotted together, displaying a grid of three plots per
+    distribution.
 
     Parameters
     ----------
     data : InferenceData
-        :class:`arviz.InferenceData` object containing the posterior/prior data.
-    kind : str
-        kind of plot to display {"latent", "observed"}, defaults to 'latent'.
-        "latent" includes {"prior", "posterior"} and "observed" includes
-        {"observed_data", "prior_predictive", "posterior_predictive"}
+        Any object that can be converted to an :class:`arviz.InferenceData` object
+        containing the posterior/prior data. Refer to documentation of
+        :func:`arviz.convert_to_dataset` for details.
+    kind : {"latent", "observed"}, default "latent"
+        kind of plot to display The "latent" option includes {"prior", "posterior"},
+        and the "observed" option includes
+        {"observed_data", "prior_predictive", "posterior_predictive"}.
     figsize : (float, float), optional
-        Figure size. If None it will be defined automatically.
+        Figure size. If `None` it will be defined automatically.
     textsize : float
-        Text size scaling factor for labels, titles and lines. If None it will be
-        autoscaled based on ``figsize``.
-    var_names : str, list, list of lists
-        if str, plot the variable. if list, plot all the variables in list
-        of all groups. if list of lists, plot the vars of groups in respective lists.
+        Text size scaling factor for labels, titles and lines. If `None` it will be
+        autoscaled based on `figsize`.
+    var_names : list of str, optional
```

Comment:
I think this is `str, list of str or list of list of str`. the "if list of lists, plot the vars of groups in respective lists." might have been an attempt to explain that. I think some kind of list of list is needed because often the variables in observed_data and posterior_predictive do not have the same name for example. It should have an example too though. You can load an inferencedata with the same names and use `idata.posterior = idata.posterior.rename(defs="defs_post")` to get different names and show usage.

Not sure if it should be `[["defs_post", "defs"]]` or `[["defs_post"], ["defs"]]` but one of the two should work